package.json

```diff
@@ -27,6 +27,7 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "autoprefixer": "^10.4.0",
+    "clean-webpack-plugin": "^4.0.0",
```

Comment:
I'm curious what this does / why we need it. Perhaps @jpjpjp can weigh in here?

Reply:
When you run `yarn watch` it generates a ton of *.hot-update.[js,json] files in the project's dist directory.   clean-webpack-plugin automatically removes these the next time you run `yarn build`